stable/chartmuseum/ci/ingress-values.yaml

```diff
@@ -0,0 +1,9 @@
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true"
+  hosts:
+    - name: chartmuseum.domain1.com
+      path: /
+      tls: false
```

Comment:
Can we remove this file from the PR?

Reply:
We can remove it, but it's the unit test to ensure this regression doesn't happen again so I thought to include it. Do you prefer not to cover changes with tests?

Reply:
@unguiculus, would you mind reviewing this for test/CI compatibility?

Reply:
@jdolitsky, I was reading this repo's review guidelines and noticed that including this file is actually required by the guidelines:
https://github.com/helm/charts/blob/9402a760e57b24d503d8af75b35be2e91faa9f7a/REVIEW_GUIDELINES.md#tests
> This directory MUST exist with at least one test file in it.

cc @helm/charts-maintainers